app/assets/stylesheets/govuk_publishing_components/components/_organisation-logo.scss

```diff
@@ -62,7 +61,12 @@
   font-family: "HelveticaNeue", "Helvetica Neue", "Arial", "Helvetica", sans-serif;
 
   &:hover {
-    text-decoration: underline;
+    color: $govuk-link-hover-colour;
```

Comment:
Doesn't quite pull through as expected (after > before)

![Jun-01-2021 14-55-08](https://user-images.githubusercontent.com/71266765/120335468-66a21980-c2e9-11eb-8fa7-9776af76b3e8.gif)

Reply:
May need to have a chat about this. It works as expected for me.

Reply:
✅